modules/communication_module.R

```diff
@@ -131,16 +138,26 @@ communication_server <- function(id,
                  # We assume that each station has only 1 location. Or we plot all, we don't distinguish location time
                  # TODO create a function or reactive to make this selection which locations to use
                  get_stations_total <- reactive({
+                    
+                   update_data()
```

Comment:
This gives an error (silent error) when loading the tool, because them doesn't exist, I guess. For now I think it isn't any problem, just to know for later.